config/devices/amazon-firetvstick-cordova.json

```diff
@@ -0,0 +1,118 @@
+{
+  "pageStrategy": "cordova",
+  "modules": {
+    "base": "antie/devices/browserdevice",
+    "modifiers": [
+      "antie/devices/anim/noanim",
```

Comment:
Wouldn't we also need to to modify the existing FireTV config to re-enable animation?